scripts/deploy_example_chart_multi.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
I think we should update current scripts instead of created "_multi.sh" version for each of them. Could we extend our scripts to support list in "TEST_MODE" env var? E.g. now it works like TEST_MODE=aws, let's update it to support TEST_MODE=aws,azure... etc.